cos-fleet-catalog-kamelets/rename-field/pom.xml

```diff
@@ -9,26 +9,11 @@
     </parent>
 
     <modelVersion>4.0.0</modelVersion>
-    <artifactId>cos-connector-jsonpath</artifactId>
+    <artifactId>cos-connector-rename-field</artifactId>
```

Comment:
I would put all those common action into a module hence, I don't think we need a repository per action.

For some other kamelet there is a module per kamelet or group or kamelet because changing i.e. and aws kamelet should not require the azure connectors to be rebuild. In this case, since all the kamelets are meant to be utility functions, a change requires also a rebuild of the connectors.

Reply:
OK will do - is there anything else we want to put into common together? log, serdes?

Reply:
let start with this one only, we must cleanup a bunch of stuffs after this pr gets merged

Reply:
Done